bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/config/YamahaZoneConfig.java

```diff
@@ -38,7 +42,7 @@
      */
     private float volumeDbMax = 12f; // 12.0 dB
 
-    public YamahaReceiverBindingConstants.Zone getZone() {
+    public YamahaReceiverBindingConstants.@Nullable Zone getZone() {
```

Comment:
That looks weird. Any possibility to move the `@Nullable` to the front?

Reply:
The only way would be to statically import Zone from YamahaReceiverBindingConstants I guess